cmd/convox/apps.go

```diff
@@ -6,8 +6,9 @@ import (
 	"strings"
 	"time"
 
+	cli "gopkg.in/urfave/cli.v1"
+
 	"github.com/convox/rack/cmd/convox/stdcli"
-	"gopkg.in/urfave/cli.v1"
```

Comment:
Didn't notice that. And it also compiled fine because `cli` is the actual pkg name. Wonder why goimports did this.